src/components/activity/FormSetDates.js

```diff
@@ -0,0 +1,64 @@
+import {useState} from 'react';
+
+function FormSetDates({addActivity}) {
+    const [activity, setActivity] = useState({
+        activity: "",
+        startTime: "00:00",
+        endTime: "00:00",
+      });
+
+    const handleChange = (event) => {
+        setActivity({ ...activity, [event.target.name]: event.target.value });
+      };
+    
+    const handleSubmit = (event) => {
+    event.preventDefault();
+    addActivity(activity);
+    setActivity({  activity: "", startTime: "00:00", endTime: "00:00", });
+    };
+   
+    return (
+        <div className="formContainer">
```

Comment:
When using Styled Components, you should not use classes from GlobalStyles but instead style your components with the `styled` function. That way you have the benefits of

- encapsulating your styles with your components
- using props to dynamically change styles

E.g. instead of `<div className="formContainer">` you could declare a `<StyledFormContainer>` component.